update.go

```diff
@@ -184,8 +183,15 @@ other options are ignored.
 		config.Cgroups.Resources.KernelMemoryTCP = int64(*r.Memory.KernelTCP)
 		config.Cgroups.Resources.Memory = int64(*r.Memory.Limit)
 		config.Cgroups.Resources.MemoryReservation = int64(*r.Memory.Reservation)
-		config.Cgroups.Resources.MemorySwap = int64(*r.Memory.Swap)
```

Comment:
@cyphar @hqhq 
I have made the changes to int64 and made it consistent for all memory operations. Let me know if you are OK with the changes.